bin/hdfcoinc/pycbc_multiifo_coinc_findtrigs

```diff
@@ -0,0 +1,351 @@
+        trigger_id: numpy.ndarray
+            The indices of this templates triggers
+        """
+        self.template_num = num
+        times = self.get_data('end_time', num)
+
+        # Determine which of these template's triggers are kept after
+        # applying vetoes
+        if self.valid:
+            self.keep = veto.indices_within_times(times, self.valid[0], self.valid[1])
+            logging.info('applying vetoes')
+        else:
+            self.keep = numpy.arange(0, len(times))
+
+        if self.bank is not None:
+            self.param = {}
+            if 'parameters' in self.bank.attrs :
+                for col in self.bank.attrs['parameters']:
+                    self.param[col] = self.bank[col][self.template_num]
+            else :
+                for col in self.bank:
+                    self.param[col] = self.bank[col][self.template_num]
+
+        # Calculate the trigger id by adding the relative offset in self.keep
+        # to the absolute beginning index of this templates triggers stored
+        # in 'template_boundaries'
+        trigger_id = self.keep + self.file['%s/template_boundaries' % self.ifo][num]
+        return trigger_id
+
+    def __getitem__(self, col):
+        """ Return the column of data for current active template after
+        applying vetoes
+
+        Parameters
+        ----------
+        col: str
+            Name of column to read
+
+        Returns
+        -------
+        data: numpy.ndarray
+            The requested column of data
+        """
+        if self.template_num == None:
+            raise ValueError('You must call set_template to first pick the '
+                             'template to read data from')
+        data = self.get_data(col, self.template_num)
+        data = data[self.keep] if self.valid else data
+        return data
+
+logging.info('Starting...')
+
+num_templates = len(h5py.File(args.template_bank, "r")['template_hash'])
+tmin, tmax = parse_template_range(num_templates, args.template_fraction_range)
+logging.info('Analyzing template %s - %s' % (tmin, tmax-1))
+
+# Create dictionary for trigger files
+trigs={}
+for i in range(len(args.trigger_files)):
+    logging.info('Opening trigger file %s: %s' % (i,args.trigger_files[i]))
+    trigs["trigs{0}".format(i)]=ReadByTemplate(args.trigger_files[i],
+                                               args.template_bank,
+                                               args.segment_name,
+                                               args.veto_files)
+
+coinc_segs = trigs['trigs0'].segs
+for i in range(len(args.trigger_files)-1):
+    coinc_segs = (coinc_segs & trigs['trigs{0}'.format(i+1)].segs).coalesce()
+
+
+for i in trigs:
+    trigs[i].segs = coinc_segs
+    trigs[i].valid = veto.segments_to_start_end(trigs[i].segs)
+
+# initialize a Stat class instance to calculate the coinc ranking statistic
+rank_method = stat.get_statistic(args.ranking_statistic)(args.statistic_files)
+if args.use_maxalpha:
```

Comment:
@tdent Did you add this? Is this needed? It looks like it's specific to a single ranking statistic.

Reply:
No, and I don't know, and I don't know what is happening here.

Reply:
@tapaimarton This probably is not the right way to pass this kind of option, so please take it out anyway. We can add it back if we remember why it was needed (we never used it in any production runs).